katran/lib/bpf/balancer_consts.h

```diff
@@ -25,6 +25,11 @@
 #define BE_ETH_P_IP 8
 #define BE_ETH_P_IPV6 56710
 
+
+// GUE varian 1 using first four bits of inner packet as a pseudo header
+// we are using last two of this four bits to distinct v4 vs v6. see RFC for more details
+#define GUEV1_IPV6MASK 0x30
```

Comment:
Also, so that we don't forget - this mask https://tools.ietf.org/html/draft-ietf-intarea-gue-05?fbclid=IwAR39fGS_slDK4PFeg1uTTcE8dGJ79AhgEJjnkQ1xm7U0IISNXYMX-l5oQ2I#section-4

Reply:
actually mask is corrent. we want to know is it v6 or v4. for v4 first 4 bits are:
0100
for v6:
0110

our mask is 0011 (+0000; but we only care about first 4 bits)

v4 & mask == 0
v6 & mask != 0. so it is working as intended